content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -123,6 +123,7 @@ operator.
   you implement yourself
 * [Operator Framework](https://operatorframework.io)
 * [shell-operator](https://github.com/flant/shell-operator)
+* [Mast](https://docs.ansi.services/mast/user_guide/operator/) (Dynamic Declarative Operator for Ansible)
```

Comment:
This list is now not in alphabetical order.

Please read the [information](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/#third-party-content-disclaimer) about editing that page.

Also, Mast is not an abbreviation for “Dynamic Declarative Operator for Ansible” so the text inside parens is not needed.

Reply:
Ok, agree with order - sorry for this.
About parens - not all the comments there describe abbreviation, metaoperator comment describe what it can do - adding here what technology mast use will be helpful, especially when the name doesn't fit into some pattern that will suggest that.

Reply:
Of course, is only suggestion - we can change it to (Ansible) or remove.

Reply:
Remove, please.

Reply:
okay

Reply:
Done - we will need to squash this commits - hope GitHub can do it automatically without my manual work ;)